src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMailboxes.scala

```diff
@@ -5,6 +5,10 @@ import io.gatling.http.Predef._
 import io.gatling.http.check.HttpCheck
 import org.apache.james.gatling.utils.JmapChecks
 import org.apache.james.gatling.utils.RetryAuthentication.execWithRetryAuthentication
+import org.apache.james.gatling.utils.RandomStringGenerator
+
+case class Id(id: String = RandomStringGenerator.randomString)
```

Comment:
you should not use the default parameter for that but use a factory method instead